programs/mango-v4/src/state/bank.rs

```diff
@@ -127,10 +127,16 @@ pub struct Bank {
     /// See scaled_init_asset_weight().
     pub deposit_weight_scale_start_quote: f64,
 
+    // We have 3 modes
+    // 0 - Off,
+    // 1 - NoDepositsNoBorrows - standard
+    // 2 - NoBorrows - borrows can only be reduced, but deposits have no restriction, special case for
```

Comment:
minor: call this ReduceDepositsReduceBorrows and ReduceBorrows? You can still deposit to repay etc...